### PR TITLE
Refactor event methods in main module and add tests.

### PR DESCRIPTION
**Pull request by josemauro**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -75,39 +75,43 @@ def rest_get(self, namespace, start=None, end=None, method=None,
...
+
         try:
             self.backend.save(event.content['namespace'],
                               event.content['value'],
                               event.content['timestamp'])
-            result = 'Value saved.'
-            error = None
         except (NamespaceError, ValueConvertError) as exc:
-            result = None
             error = (str(exc), exc.__class__.__name__)
```
Comment (ajoaoff) [inline main.py:86]: With this change, `result` is always `None`. It should be set when successfull.

Comment (josemauro) [reply main.py:86]: Which value it should be set?

Comment (ajoaoff) [reply main.py:86]: Just a message reporting success.

Comment (josemauro) [reply main.py]: I added the message to save method and delete. Thanks.

File: main.py [hunk 2]
```diff
@@ -75,39 +75,45 @@ def rest_get(self, namespace, start=None, end=None, method=None,
...
             self.backend.delete(event.content['namespace'],
-                                event.content['timestamp'])
-        except Exception as exc:
-            result = None
-            error = (exc.__class__, exc.args)
+                                event.content['start'],
+                                event.content['end'])
+            result = 'Value deleted.'
+        except (NamespaceError, ValueConvertError, ValueError) as exc:
+            error = (str(exc), exc.__class__.__name__)
```
Comment (hdiogenes) [inline main.py:116]: I suggest reversing the order of the parameters to be more like a Python exception message, with the class name first:
`ValueError: value XYZ is invalid`

**Pull request merged**